.travis.yml

```diff
@@ -25,7 +25,18 @@ matrix:
           packages:
             - libaio1
             - rpm
+      sudo: required
+    - env:
+        - ORACLE_HOME=/home/travis/oracle/u01/app/oracle/product/11.2.0/xe
+          LD_LIBRARY_PATH=/home/travis/oracle/u01/app/oracle/product/11.2.0/xe/lib
+          ORACLE_SID=XE
+      sudo: false
+      addons:
+        apt:
+          packages:
+            - libaio1
+            - rpm
 
 before_install:
-  - ./download.sh
-  - ./install.sh
+  - bash -x ./download.sh
```

Comment:
these changes are only to help reviewing what is happening under the hood using the log output